libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -1642,5 +1642,11 @@ uint8_t AP_AHRS_NavEKF::get_primary_gyro_index(void) const
     return AP::ins().get_primary_gyro();
 }
 
+
+AP_AHRS_NavEKF &AP::ahrs_navekf()
+{
+    return static_cast<AP_AHRS_NavEKF&>(*AP_AHRS::get_singleton());
```

Comment:
This is ugly as hell. I'll need to start pressuring you to make the AHRS_NavEKF be the AHRS interface 😃 - and it should really be just an interface that AHRS_NavEKF and AHRS_View implement (and DCM be independent with an instance inside AHRS_NavEKF).